pages/sign-up.js

```diff
@@ -1,11 +1,32 @@
+import Link from "next/link";
 import { motion as m } from "framer-motion";
+import { AiOutlineTwitter } from "react-icons/ai";
 
-const singUp = () => {
+const signUp = () => {
```

Comment:
Can we change the first letter to be Capitalize?

Reply:
in javaScript we mostly use camel case for naming thee variables i think

Reply:
It is absolutely true but that is for function.
but when we are making a component, We have to use capitalize words, it's a rule.